Replicator/Checkpointer.cc

```diff
@@ -233,9 +234,18 @@ namespace litecore { namespace repl {
         // Existing documents of are considered in the default collection
```

Comment:
existing documents of what?

Reply:
Documents that existed prior to a 3.1 upgrade